README.md

```
@@ -15,64 +15,83 @@
 
 A Swift Package for Server-Side and Command-Line Access to CloudKit Web Services
 
-## Overview
+## Table of Contents
+- [Overview](#overview)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)